### ISSUE TITLE
Mike passed away

### ISSUE THREAD

**Issue by peterLaurence** [moagrius/TileView#540]
Hello, I'm Pierre, a contributor on this project. 

It is with great sadness that I have to inform you that Mike Dunn (@moagrius ), the maintainer of TileView, has passed away. Mike and I worked together on TileView since late 2015. We've spent countless hours on code reviews.

This is truly a great loss for everyone and the open source community. Mike, I'll never forget you.

**Comment by egepointr** [comment]
:/ Our condolences from team Pointr

**Comment by subzthanabalan** [comment]
Thank you for this post, @peterLaurence. 

Mike was a dear friend and mentor to me at O'Reilly.  This has been such a great loss for us all and we are truly devastated. Please feel free to reach out to me if you need someone to talk to or would like to share stories about Mike.

I am an Android engineer and would love to ensure that TileView remains actively supported. I plan on contributing to the project as much as possible.

**Comment by peterLaurence** [comment]
Hi @subzthanabalan,

We were writing a book together and were about half way. For now, O'Reilly and I agreed to dedicate this book to him - of course it will take more time for me to finish this alone.

I met Mike while I was a junior Android developer looking for a nice library to display maps for my trekking application. As I was looking for good performances for my app, I had to contribute to TileView as well. Long story short, I ended up knowing the internal plumbings of TileView (v2) as well as Mike. That was in 2016.
Last year, Mike released TileView v4, which is a major re-write of v2 (v3 wasn't mature enough and has never been official). TV4 adds better memory management with an internal LRU cache, tile subsampling when zoomed-out, among other things.
Meanwhile, like many Android engineer I was discovering Kotlin and was completely mind blown. We knew that re-writing TileView in idiomatic Kotlin was a massive undertaking. So we never seriously considered it. I don't know why, one day I wanted to set up a proof of concept of a TileView in Kotlin using coroutines. It turned out to have amazing performance, so I carried on and two month later I told Mike that I did it. He was really enthusiast about it. Back then, it was a just a package in my Android app because I didn't want to take over. I left it like that and was happy about it. But shortly after, Mike encouraged me to release this as my own project. So I had to do pretty the same thing as for TileView: a sample app along with the library itself with all implications like proper documentation, publication, etc. If Mike hadn't pushed me to do this, the [MapView](https://github.com/peterLaurence/MapView) project would never exist. He even added a banner to his own repo, suggesting Kotlin's fans to give MapView a try. Mike was a truly open source believer.

So basically, MapView is TileView in Kotlin. This is the project I actively maintain. I would never discourage anyone from contributing to TileView - and as a contributor, I will take care of PRs. I just want to say that I'm focusing on MapView since Kotlin is my #1 langage now. In a sense, MapView is also Mike's project.

**Comment by ehorvat** [comment]
@peterLaurence I wanted to extend my thanks as well, and echo what @subzthanabalan has already said. Mike was an incredible mentor and had such a wealth of knowledge. You could not have asked for a better team member.

**Comment by ControlAndC** [comment]
That's a very sad news, I used TileView as a life saver in my project.
 May his soul in rest and peace
